tst/EnergyPlus/unit/HVACStandaloneERV.unit.cc

```diff
@@ -0,0 +1,95 @@
+// EnergyPlus::Standalone ERV Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// EnergyPlus Headers
+#include <EnergyPLus/HVACStandAloneERV.hh>
+#include <EnergyPlus/DataHeatBalance.hh>
+#include <EnergyPlus/DataZoneEquipment.hh>
+#include <EnergyPlus/DataSizing.hh>
+#include <EnergyPlus/UtilityRoutines.hh>
+#include <ObjexxFCL/gio.hh>
+#include <EnergyPlus/SQLiteProcedures.hh>
+
+
+using namespace EnergyPlus;
+using namespace EnergyPlus::HVACStandAloneERV;
+using namespace ObjexxFCL;
+using namespace EnergyPlus::DataHeatBalance;
+using namespace DataGlobals;
+using namespace EnergyPlus::DataZoneEquipment;
+using namespace EnergyPlus::DataSizing;
+
+
+TEST( SizeStandAloneERVTest, Test1 )
+{
+
+	int write_stat;
+	// Open the Initialization Output File (lifted from SimulationManager.cc)
+	OutputFileInits = GetNewUnitNumber( );
+	{ IOFlags flags; flags.ACTION( "write" ); flags.STATUS( "UNKNOWN" ); gio::open( OutputFileInits, "eplusout.eio", flags ); write_stat = flags.ios( ); }
+//	eso_stream = gio::out_stream( OutputFileStandard );
+
+	//CreateSQLiteDatabase(); (lifted from SimulationManager.cc)
+	try {
+		EnergyPlus::sqlite = std::unique_ptr<SQLite>( new SQLite( ) );
```

Comment:
@mjwitte No you don't need to do anything. There is no sqlite database created or used. Since it is a unique_ptr it should clean up itself...

[std::unique_ptr is a smart pointer that retains sole ownership of an object through a pointer and destroys that object when the unique_ptr goes out of scope.](http://en.cppreference.com/w/cpp/memory/unique_ptr)